staging/src/k8s.io/apiserver/pkg/server/config.go

```diff
@@ -233,6 +234,9 @@ type Config struct {
 
 	// StorageVersionManager holds the storage versions of the API resources installed by this server.
 	StorageVersionManager storageversion.Manager
+
+	// RequestWidthEstimator is used to estimate the "width" of the incoming request(s).
+	RequestWidthEstimator flowcontrolrequest.WidthEstimatorFunc
```

Comment:
Should this be in the section of fields that are targets for removal?

Reply:
It shouldn't.

BTW - I think that a high-level comment "for removal" is very easy to miss (I missed that). I would much rather prefer having individual fields tagged as "deprecated" or 'for removal" or sth.
[Not for this PR of course.]

Reply:
still not address (please just move to above)